torch/csrc/jit/passes/onnx/preinline_onnx.cpp

```diff
@@ -0,0 +1,61 @@
+#include <torch/csrc/jit/passes/onnx/preinline_onnx.h>
+#include <torch/csrc/jit/jit_log.h>
+#include <torch/csrc/jit/passes/onnx/helper.h>
+
+namespace torch {
+namespace jit {
+
+const auto namespaceName = std::string("___torch_mangle_3");
+const auto qualFuncName =
+    std::string("__torch__.torch.nn.functional.") + namespaceName + ".";
+
+void PreInlineCalls(Block* block) {
+  for (auto it = block->nodes().begin(), end = block->nodes().end();
+       it != end;) {
+    Node* cur = *it++;
+    switch (cur->kind()) {
```

Comment:
nit: there seems to be only one real case for this `switch`. Would making it an (if, else) be clearer? Assuming we do not anticipate adding more cases in the near future.

Reply:
I'm going to check what's prim::CallMethod and whether we want to have that. If not, I'll remove this.

Reply:
Actually Call Method is for calling builtin methods. It's not needed for this pass.